spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/GravitinoCatalog.java

```diff
@@ -336,4 +351,27 @@ private String getDatabase(NameIdentifier gravitinoIdentifier) {
         "Only support 3 level namespace," + gravitinoIdentifier.namespace());
     return gravitinoIdentifier.namespace().level(2);
   }
+
+  @VisibleForTesting
```

Comment:
could you move corresponeding test class to the same package name under test dir? then we could remove public here

Reply:
fixed